module/Status/src/V1/Rest/Status/StatusResourceFactory.php

```diff
@@ -1,12 +1,12 @@
 <?php
 namespace Status\V1\Rest\Status;
 
-use StatusLib\MapperInterface;
+use StatusLib\ArrayMapper;
 
 class StatusResourceFactory
 {
     public function __invoke($services)
     {
-        return new StatusResource($services->get(MapperInterface::class));
+        return new StatusResource($services->get(ArrayMapper::class));
```

Comment:
I think something changed in the Zend module... Since I'm not that familiar with Zend Modules (nor Apigility) I kind of had to figure this out. Luckily this was an easy fix.

As mentioned in #1 ; I have to do some more reading into Zend Modules and alias configuration

https://apigility.org/documentation/intro/first-rest-service#defining-the-resource
 ```PHP
<?php
namespace Status\V1\Rest\Status;

use StatusLib\Mapper;

class StatusResourceFactory
{
    public function __invoke($services)
    {
        return new StatusResource($services->get(Mapper::class));
    }
}
```

I assume `StatusLib\Mapper` being an alias, more info might be found here https://github.com/zfcampus/statuslib-example#configuration haha